LPFilter/Source/PluginProcessor.h

```diff
@@ -71,12 +71,9 @@ class LpfilterAudioProcessor  : public AudioProcessor
     AudioParameterChoice* mode;
     AudioParameterBool* bypass;
     IIRCoefficients      iirCoef;
```

Comment:
We use two different IIR coefficients.
Is this done because the custom filter and juce-dspFilters don't have the same order?

Reply:
I just didn't want to use the `juce_dsp` module for both custom filter and the lpfJuce filter. So I found the class `IIRCoefficients` from `juce_audio_basics`. And that's the reason I get a different order for the custom filter.